### DISCUSSION TITLE
OpenVINO™ Noise Suppression : Error: Failed to load shared library 'avformat-60.dll' (error 126:

### DISCUSSION THREAD

**Discussion by brainai-hub** [intel/openvino-plugins-ai-audacity#275] [Q&A] [answered]
Hello ~~

I am trying to use Intel AI PC with Audacity. I want to see how NPU works in Audacity Program. 
But I met error. Anyone can help me?

![error1-Screenshot 2024-08-23 155206](https://github.com/user-attachments/assets/63bae8fa-d20e-4162-9af1-0994a47f1134)
![error2-Screenshot 2024-08-23 155206](https://github.com/user-attachments/assets/6f78efcd-315d-4b6c-a013-2c3f2ef4e87e)
![error3-Screenshot 2024-08-23 155206](https://github.com/user-attachments/assets/16cdd4f9-acac-473a-b282-f0de3936e9f9)

**Comment by RyanMetcalfeInt8** [comment]
Hi @brainai-hub,

Could you share the entire contents of your Audacity log? Your screenshot only shows the first 8 lines.

Also, could you share the version of the NPU driver that you have installed on your system? You can open up task manager, go to 'Performance' tab, and click on 'NPU' -- in the lower right hand corner you'll see the version:
<img width="950" alt="image" src="https://github.com/user-attachments/assets/21733eb7-d26e-4127-98da-9c668dbe7acf">

**Comment by brainai-hub** [comment]
Thank you for your quick reply.

Here is my NPU version.

![image](https://github.com/user-attachments/assets/4fe87b2d-5585-45fd-8bc3-c5f118f03ec7)

**Comment by brainai-hub** [comment]
![image](https://github.com/user-attachments/assets/04f332d2-557e-45cd-9a2c-b611e1ee586d)

![image](https://github.com/user-attachments/assets/37aa8af4-a185-4d1b-94b2-ba245ca2adf0)

![image](https://github.com/user-attachments/assets/1e93c253-b652-455c-8c46-47ce53c44fbe)

![image](https://github.com/user-attachments/assets/0b78d0db-5378-47da-a1d7-dd86e39aa6c3)

![image](https://github.com/user-attachments/assets/c5e212fa-3802-4ccc-8491-cd3fc47099cd)

**Comment by RyanMetcalfeInt8** [comment, marked as answer]
It looks like your NPU driver is quite old (1688). Can you try updating to the latest version? It can be downloaded from here:
https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html

**Reply by brainai-hub** [reply]
Thanks you for your kind guide. 

I have updated NPU driver, so no more error, but still not working NPU. 
Any more advise for me to move next.

![image](https://github.com/user-attachments/assets/11ebb4fc-8a9f-4e98-99ec-8f264906bb40)

**Reply by brainai-hub** [reply]
Once turn off and turn on computer, It works. 

I think I can fix thanks to you.

Thanks a ton.


![image](https://github.com/user-attachments/assets/51f83aa1-254a-469f-8853-6a02e4af5b6d)

**Comment by brainai-hub** [comment]
Could you let me know the link where I can update Ultra arc GPU driver?

Here is my current GPU version.

![image](https://github.com/user-attachments/assets/7fddc080-0d72-4b40-9621-eb07e855635d)

**Reply by gblong1** [reply]
Core Ultra drivers can be found on this page: https://www.intel.com/content/www/us/en/products/details/processors/core-ultra/downloads.html Look for the Intel Arc and Iris Xe driver or Arc Pro driver, depending on your system.

**Reply by brainai-hub** [reply]
Thanks a lot. Your support has been a great help.

**Comment by brainai-hub** [comment]
Dear,

I don't know if you are the right contact about this issue. But I can't
find anyone to answer this issue.
Could you help me out?

https://github.com/intel/dream-textures-openvino/issues/7

This is about "Dream-textures-openvino", similar to Audacity OpenVINO.

I am trying to Install and use the Openvino Dream Textures feature.
https://github.com/intel/dream-textures-openvino

But I met the error below. I asked some educators who use Intel AI PC.
They all met the same error. But we don't know how to fix this problem.

[image: image.png]

Recently I have had a great experience with your support.
So I just wonder if I can get any help from you.

Please let me know if you have any ideas.

Thanks, Best Regards.



*John Kim*
*CEO of BrainAI Co., Ltd*

*Creating a Better World through Social Impact!*
#707, 50, Digital-ro 33-gil, Guro-gu, Seoul, Republic of Korea, 08377
Tel: +82-2-6478-8112 | Mobile: +82-2-10-4603-5399
Website: http://brainai.kr


*Confidentiality Caution*: If you are not the intended recipient of this
communication, please do not retain,
use, disclose, disseminate, distribute or copy the contents to any person,
as it may be unlawful.
